pkg/util/util.go

```diff
@@ -0,0 +1,41 @@
+package util
+
+import (
+	"context"
+	"github.com/blang/semver"
+	"github.com/google/go-github/github"
+	"github.com/pkg/errors"
+	"strings"
+)
+
+const (
+	ghorg            = "redhat-developer"
+	ghrepo           = "ocdev"
+	InstallScriptURL = "https://raw.githubusercontent.com/redhat-developer/ocdev/master/scripts/install.sh"
+)
+
+func GetNewerReleaseTag(currentVersion string) (string, bool, error) {
```

Comment:
can we have please comments describing what this function is and what it is returning

Reply:
shouldn't this be named `GetLatestReleasedTag` or `GetLatestRelease`?

Reply:
it might be also better to have a special package for this, rather than putting it into generic `util`.`updater` or `notifier` might be good names

Reply:
- added comments
- renames to `CheckLatestReleaseTag`
- changed package to `notify`
- broke the function down to 2 functions, one to get the latest release tag from GH, another is the exported function
- changed function signature not to return a bool, instead make decisions if the string is empty or not